### DISCUSSION TITLE
Requerimiento funcionales y no funcionales

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#89] [Ideas]
Buenas tardes, una pregunta¿Como cuantos requerimiento sacaron? Yo saque como 10
Funcionales
1. El formulario
2. Proceso de pago
3. Cuenta
4. Vista del cliente
5. Documento legal 
6. Vista del abogado 
7. Notificaciones de usuario
8. Pagina o sitio responsivo
9. Preferencias de colores
No funcionales
1. Tiempo de respuesta
2. Error
3. Conexión
4. Interacción con el cliente
5. Login (usuario/administrador)

**Comment by martinGM05** [comment]
Esos, igual puedes describirlos por separado en el caso de CRUD, uno especial para crear, otro para consultar, etc.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale gracias, igual seria como no funcional, aunque siento que solo seria consultar y a lo mejor editar

**Reply by AngelCruzO** [reply]
Yo normalmente pongo operaciones CRUD en funcionales, ya que es algo que hace el sistema. Y según mi definición de no funcionales es como hace esas operaciones, que es por medio de una conexión a base de datos y presentándolos en la vista

**Reply by GloriaCoralCerecedo** [reply]
Ok entiendo yo tenia entendido igual que no funcional era como algo que no te pide el usuario pero que si es importante que lleve. Algo que no esta en el proceso por ejemplo pero sirve

**Reply by AngelCruzO** [reply]
No es así, si tienes una idea que el cliente no pidió también puede ser funcional o no funcional. Normalmente los no funcionales son cosas de rendimiento, cantidad de caracteres, velocidad de procesamiento, tamaños de pantalla, etc. Bueno eso es lo que a mí me enseñaron en mi formación académica, puede que esté mal

**Reply by GloriaCoralCerecedo** [reply]
Oh entiendo, si es verdad, son cosas que en si no afectan el funcionamiento del sistema.

**Reply by AngelCruzO** [reply]
Asi es

**Comment by GerAlonso** [comment]
Entonces el formato propuesto por el Mission Comander de requerimientos se tendría que llenar y desarrollar para cada uno de esos requerimientos? Osea en teoría se realizaría 14 veces?

**Reply by AngelCruzO** [reply]
Si, pero existen algunas partes que creo solo se hacen una sola vez

**Reply by GloriaCoralCerecedo** [reply]
Creo yo que solo los requerimiento y las historias de usuarios se tendrían que realizar dependiendo de cuantas allá.
Una pregunta, ¿Las historias de usuario van conforme a los requerimiento? Osea son los mismos, parecidos o diferentes. Son 14 requerimientos por ejemplo, entonces debe de haber 14 historias de usuarios o 6  en caso diferente de que no precisamente tengan que ser lo mismo

**Reply by AngelCruzO** [reply]
Yo no he manejado historias de usuario, solo requerimientos.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale
